.github/workflows/docker-build.yaml

```
@@ -15,6 +15,10 @@ on:
         required: false
         type: string
         default: "${{ github.sha }}"
+      labels:
+        required: false
+        type: string
+        default: "git.commit=${{ github.sha }}"
```

Open question, would adding the repository as well be helpful?
```suggestion
      labels:
        required: false
        type: string
        default: |
          git.commit=${{ github.sha }}
          git.repository=${{ github.repository }}
```